wakeonlan: add page (#13424)
tldr-pages/tldr: pages/common/wakeonlan.md

# wakeonlan

> > Send packets to wake-on-lan (WOL) enabled PCs.

Reviewer: ```suggestion
> Send packets to wake-on-LAN (WOL) enabled PCs.
```

> > wakeonlan - sends 'magic packets' to wake-on-lan (WOL) enabled PCs.

Reviewer: ```suggestion
> Send packets to wake-on-lan (WOL) enabled PCs.
```
See style guide.

> > See also: `wakeonlan`.

Reviewer: ```suggestion
```
See also is not correct as `wakeonlan` is the current page.

> > More information: <https://github.com/jpoliv/wakeonlan>

Reviewer: ```suggestion
> More information: <https://github.com/jpoliv/wakeonlan>.
```

> - Send magic packets to all devices on the local network (255.255.255.255) by specifying a MAC address:

Reviewer: ```suggestion
- Send packets to all devices on the local network (255.255.255.255) by specifying a MAC address:
```

`wakeonlan {{01:02:03:04:05:06}}`

> - Send magic packet to a specific device via IP address:

Reviewer: ```suggestion
- Send packet to a specific device via IP address:
```

`wakeonlan {{01:02:03:04:05:06}} -i {{192.168.178.2}}`

- Print the commands, but don't execute them (dry-run):

`wakeonlan -n {{01:02:03:04:05:06}}`

> - quiet mode:

Reviewer 2: ```suggestion
- Run in quiet mode:
```
